texar/tf/hyperparams.py

```diff
@@ -271,7 +267,7 @@ def _parse(hparams,
                 parsed_hparams[name] = value
             elif isinstance(value, type(default_value)):
                 parsed_hparams[name] = value
-            elif is_callable(value) and is_callable(default_value):
+            elif callable(value) and callable(default_value):
```

Comment:
Can `callable` truly replace `is_callable`, which is a wrapper over `callable`? (https://github.com/asyml/texar/blob/master/texar/tf/utils/dtypes.py#L82)

Reply:
I think it is true in Python 3.